### PR TITLE
Test: Make a By message on connection test.

### PR DESCRIPTION
**Pull request by eloycoto**
Make a By message to make sure to know where the test is failing.

Try to debug issues:
https://jenkins.cilium.io/job/Ginkgo-CI-Tests-Pipeline/6100/execution/node/90/log/?consoleFull

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7267)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

File: test/k8sT/Updates.go [hunk 1]
```diff
@@ -187,9 +188,6 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 			err = kubectl.WaitForKubeDNSEntry(app1Service, helpers.DefaultNamespace)
 			ExpectWithOffset(1, err).To(BeNil(), "DNS entry is not ready after timeout")
 
-			err = kubectl.CiliumEndpointWaitReady()
```
Comment (tgraf) [inline test/k8sT/Updates.go:190]: Please state  in the commit message why removing this is needed.

**Review by tgraf — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

**Comment by eloycoto** [comment]
FQDN issue: https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/10382/testReport/junit/(root)/runtime/RuntimeFQDNPolicies_Implements_matchPattern_____/

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**